### ISSUE TITLE
Remove some unwanted packages ?

### ISSUE THREAD

**Issue by dashbaz** [ndimatteo/HULL#89]
Hi, 

First off, thank you. I've been working with this for over a week and almost done configuring it to my specific project. 

I'm running these lighthouse reports to improve the performance score. Last several tests have ranged from 65-90. I'm trying to get it high and consistent.  

Looking to remove some Facebook data showing up in the report as I won't be using it. 

![Screen Shot 2022-04-26 at 2 14 32 PM](https://user-images.githubusercontent.com/90878373/165368802-784c3e10-6a7e-4060-9e08-29f613db71e2.png)

**I've searched the repo and purged everything Facebook(&twitter) to no avail. Can you please provide insight into how I locate this and get rid of it?

Also: 

In the Wappalyzer chrome extension on my laptop, it's showing 13 technologies. For instance, styled components is in the studio folder package.json. I did a 'npm uninstall styled-components --save' in the studio, pushed it up and it still showed on laptop. 

![Screen Shot 2022-04-26 at 2 20 18 PM](https://user-images.githubusercontent.com/90878373/165366321-2d716934-d993-48c9-88e3-2e33dbac464b.png)

**On my desktop it's showing 7, which is how I prefer it. Would you happen to know why this is occurring? 

![Capture](https://user-images.githubusercontent.com/90878373/165366811-c2b243d5-1491-49d0-b7a2-3e61cde430db.JPG)

Thanks

**Comment by ndimatteo** [comment]
Hey there @dashbaz, glad you're enjoying the starter!

The only reason Facebook and Google Analytics would be showing is if you left the Google Tag Manager example in place. 

This is included in the demo content when you do the automatic setup, and can be found within the Sanity Studio under:
**Settings > General > Advanced**

When that is set, it includes Google Tag Manager in this component here: https://github.com/ndimatteo/HULL/blob/3d9ef00107a1e761bfc5ceb1b2c99e1ac80f0151/components/layout.js#L53-L64

If you clear that value in the CMS and redeploy you shouldn't see any of those 3rd-party scripts appearing.

As for `styled-components` that's needed for the Studio build. I'm not familiar with the Wappalyzer extension, so I'm not sure exactly what it's flagging there. I do see that it specifies that it's for development, so you can likely ignore this. Again, it's only used for the Studio side of things, so your front-end should not be affected by it.

**Comment by dashbaz** [comment]
It worked, thanks. 

Wappalyzer analyzes the tech used on a website. I removed it from the CMS and frontend. 
It persisted in showing facebook, etc in the tech list until I cleared the cache in the Wappalyzer extension itself. 

I assumed I could decouple the facebook and gtm as I plan on implementing the gtm for use on my website. I will look for ways to implement it in the frontend like a regular react/next app once I wrap up the styling.

Appreciate everything!
